daemon/stop.go

```diff
@@ -23,6 +23,9 @@ func (daemon *Daemon) ContainerStop(name string, seconds *int) error {
 	}
 	if !container.IsRunning() {
 		err := fmt.Errorf("Container %s is already stopped", name)
```

Comment:
In addition, from my point of view, maybe our taking two steps is just enough:
1. change the error message from `Container %s is already stopped` to `Container %s is not running`;
2. make the status code of 304 and the message above received by client side.

@yongtang WDYT?